widgets/fileupload/assets/js/focuspoint-tool.js

```diff
@@ -0,0 +1,90 @@
+window.addEventListener('load', function () {
+    initFocuspointTool()
+    let observableNode = document.querySelector('.upload-files-container') ? document.querySelector('.upload-files-container') : document.querySelector('.rowlink')
```

Comment:
Don't use ES6 features for better browser support (whole file)!

* `let` / `const` => `var`
* `() => {}` => `function() {}`